README.md

```diff
@@ -1 +1,43 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### Introduction
+
+My name is Vanessa Guigon and I am from *Chicago, Illinois*. I
```

Comment:
Not sure why this indentation looks the way it does but you don't need to add extra spaces in markdown.